Commands/Commands/queue.js

```diff
@@ -370,6 +370,9 @@ commands.registerVote = {
                     cause: 'chat_vote',
                     message: (e.message !== undefined) ? e.message : JSON.stringify(e)
                   }, e)
+                } else if (e.statusCode === 422) {
```

Comment:
Maybe after this pr gets merged we look into refactoring the mess that's been created? (Specifically the huge if and error code)